src/model/event.ts

```diff
@@ -30,11 +31,42 @@ export const isPastEvent = (event: IEvent): boolean =>
 export const isComingEvent = (event: IEvent): boolean =>
   DateTime.fromISO(event.date) >= DateTime.now();
 
-export const filterPastEvents = (events: IEvent[]) =>
-  events.filter(isPastEvent);
+export const isPublished = (event: IEvent): boolean =>
```

Comment:
just FYI
the triple equal operator ensures the two operands have the same value and they are the same type (it should be always preferred over the double operator)
![Screenshot 2023-12-27 alle 20 03 43](https://github.com/latina-in-tech/latina-in-tech.github.io/assets/822471/124d5aab-9235-411d-bff2-8ab0a73329f5)